tests/jsx_fragment/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,50 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`fragment.js 1`] = `
+<></>;
+
+<>
+   text
+</>;
+
+<>
+  <Component />
+  <Component />
+</>;
+
+</* open fragment */>
+  <Component />
+  <Component />
+</ /* close fragment */>;
+
+< // open fragment
+>
+  <Component />
+  <Component />
+</ // close fragment
+>;
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<></>;
+
+<>text</>;
+
+<>
+  <Component />
+  <Component />
+</>;
+
+</* open fragment */>
+  <Component />
+  <Component />
+<//* close fragment */>;
+
+<
+  // open fragment
+>
+  <Component />
+  <Component />
+</
+  // close fragment
```

Comment:
For a self-closing JSX tag, we would put the "closing slash" after the comment or attributes. Maybe we should do the same thing for fragments?

**Prettier 1.8.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAJgSwG4B0oAEBA9AFQEBmEEBpx+xAfANz74Y74ECGAvMNwC+XAEb8ATnG4AbaQE9JM+YtkKpqlcvVala3Zr2rp0AObCoTVlBAAaEBAAOMTNADOyUN1cxkFGa7g7EXFuMABrOBgAZQdQzCgTZBhxAFdAkHQIMF9-dPiA8RgABRCTAFtuHOkAuwArVwAPACEQ8Mio7jK4ABl4uCqa+xSYB2GAJgH02PEC5BARbhE5Y2s7B3F4mAB1THQYAAtkAA4ABjXxCACtkIck1PTJAEcUzEkS7nLKpD9q9ICyzB3NJ2VzxEzSOAARRSEHgkzsMEWOz2hyQYwRIUw0jBAGEIGUKnMoNB+nYUgEACqLdzfXKCQRAA)

**Input:**
```jsx
<div
  /* foo */
/>;

<div
 a={a}
 b={reallyreallyreallyreallyreallyreallyreallyreallyreallyreallylong}
/>;

```

**Output:**
```jsx
<div
/* foo */
/>;

<div
  a={a}
  b={reallyreallyreallyreallyreallyreallyreallyreallyreallyreallylong}
/>;

```

Reply:
I was thinking more of this case:

**Prettier 1.8.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAJgSwG4D4A6UABEQCpwAeMhqA9FtkbQFRGQC27CMRztuIADQgIABxiZoAZ2SgAhlJjIAZnIA2UuMIBGAJzlgA1nBgBlUQcxQA5shi6ArlpDoIYFes3Crm3TAAK+tbsch4azgBWUhQAQvpGJqZyXAAyVnBhXiIOMKI5AEyZzha6vsgg2nLaAJ5q0EIgorpWMADqmOgwABbIABwADMJNEJqt+qJ2js66cACODpgzgXLBoUiq4cKa7JiTTltW1mpwAIoOEPBFwjBV7Z09SPnX+phqhwDCEJxrIFDQGcIHJpSFUZOtPHAAL6QoA)

**Input:**
```jsx
<div>
  Text
</div /* comment */>
```

**Output:**
```jsx
<div>Text</div /* comment */>;

```

Reply:
Since you can't put attributes in a jsx fragment (afaik), I guess it only affects comments, so it's probably fine either way